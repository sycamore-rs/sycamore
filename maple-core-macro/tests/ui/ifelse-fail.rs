use maple_core::prelude::*;

fn compile_pass() {
    template! {
        div {
            @if unknown_condition {
                p
            }
        }
    };

    template! {
        div {
            @else true {
                p
            }
        }
    };
}

fn main() {}
