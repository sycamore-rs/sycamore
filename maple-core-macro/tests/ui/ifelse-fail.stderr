error: expected `if`
  --> $DIR/ifelse-fail.rs:14:14
   |
14 |             @else true {
   |              ^^^^

error[E0425]: cannot find value `unknown_condition` in this scope
 --> $DIR/ifelse-fail.rs:6:17
  |
6 |             @if unknown_condition {
  |                 ^^^^^^^^^^^^^^^^^ not found in this scope
